staging/src/k8s.io/cri-api/pkg/apis/testing/fake_runtime_service.go

```diff
@@ -194,6 +194,12 @@ func (r *FakeRuntimeService) RunPodSandbox(config *runtimeapi.PodSandboxConfig,
 			Network: &runtimeapi.PodSandboxNetworkStatus{
 				Ip: FakePodSandboxIPs[0],
 			},
+			// kubeGenericRuntimeManager's podSandboxChanged will always recreate sandbox without setting Namespace options.
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/kuberuntime_manager.go#L445

Reply:
@gongguan could you elaborate more about the comment?

Reply:
addressed, is it ok now?

Reply:
> @gongguan could you elaborate more about the comment?

comment updated, do you think it is ok?